cdc/processor/pipeline/sink.go

```diff
@@ -136,17 +143,35 @@ func (n *sinkNode) flushSink(ctx context.Context, resolved model.ResolvedTs) (er
 			err = n.stop(ctx)
 		}
 	}()
+
 	currentBarrierTs := atomic.LoadUint64(&n.barrierTs)
-	currentCheckpointTs := n.getCheckpointTs()
-	if resolved.Ts > currentBarrierTs {
-		resolved = model.NewResolvedTs(currentBarrierTs)
-	}
 	if resolved.Ts > n.targetTs {
 		resolved = model.NewResolvedTs(n.targetTs)
 	}
+
+	if n.redoManager != nil && n.redoManager.Enabled() {
+		// redo log do not support batch resolve mode
+		resolved = model.NewResolvedTs(resolved.ResolvedMark())
+		err = n.redoManager.FlushLog(ctx, n.tableID, resolved.Ts)
+
+		redoTs := n.redoManager.GetMinResolvedTs()
```

Comment:
Should we use this value to drive sink?

Reply:
The resolvedTs used to drive Sink is only directly limited by barrierTs. In detail, the happens-before relation is:
`flushLog(resolvedTs from sorter) -> table sinkNode resolvedTs -> globalResolvedTs -> barrierTs`

Reply:
Got it. Thanks!